scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -11,6 +11,7 @@ export KUBEADMIN_PASSWORD=`cat $KUBEADMIN_PASSWORD_FILE`
 # show commands
 set -x
 export CI="prow"
+export PRIVATE_REPO_DRIVER="github"
```

Comment:
`gitlab` is mentioned in PR's title, but the driver is set to `github`. Is this a mistake?

Reply:
In CI we have configured the test run only for github repos, so i used this ```export PRIVATE_REPO_DRIVER="github"``` otherwise our CI test job will fail.

To run local test on gitlab repo:
```
$ export SERVICE_REPO_URL=<Provide the URL for your Service repository>
$ export GITOPS_REPO_URL=<Provide the URL for your GitOps repository>
$ export IMAGE_REPO=<Image repository which is used to push newly built images>
$ export DOCKERCONFIGJSON_PATH=<Filepath to config.json which authenticates the image push to the desired image registry>
$ export GITHUB_TOKEN=<Used to authenticate repository clones, and commit-status notifications (if enabled)>
$ export PRIVATE_REPO_DRIVER=gitlab
$ make e2e-local
`

Reply:
I will create a separate pr to enable test on gitlab repo in OpenShift CI